### PR TITLE
Add Features in Template.CSharp project.

### PR DESCRIPTION
**Pull request by chenzhitong**

### CODE REVIEW

File: templates/Template.CSharp/Contract1.cs [hunk 1]
```diff
@@ -5,7 +5,8 @@
 
 namespace $safeprojectname$
 {
-	public class Contract1 : SmartContract
+    [Features(ContractFeatures.HasStorage)]
+    public class Contract1 : SmartContract
```
Comment (Tommo-L) [inline templates/Template.CSharp/Contract1.cs]: Should we set the default value is `ContractFeatures.HasStorage` to replace this?

Comment (shargon) [reply templates/Template.CSharp/Contract1.cs]: I don't think so, it's an extra feature

Comment (Tommo-L) [reply templates/Template.CSharp/Contract1.cs]: We should replace here by extra features about `author`, `email`, `date` fields, as we need developers to update it. And the default value of features value is already `ContractFeatures.HasStorage`.

I think we should keep contract code as clean as possible, move configurations to `mainfest.json` as we have a lot of items to set.

**Comment by vncoelho** [comment]
Perhaps we need to clean this folder still, right? There are two csprojec

**Pull request merged**